robottelo/utils/sso.py

```diff
@@ -0,0 +1,197 @@
+import json
+import random
+from contextlib import contextmanager
+from functools import lru_cache
+
+from broker.hosts import Host
+from fauxfactory import gen_string
+from pexpect import pxssh
+
+from robottelo.config import settings
+from robottelo.constants import KEY_CLOAK_CLI
+from robottelo.constants import RHSSO_NEW_GROUP
+from robottelo.constants import RHSSO_NEW_USER
+from robottelo.constants import RHSSO_RESET_PASSWORD
+from robottelo.constants import RHSSO_USER_UPDATE
+from robottelo.datafactory import valid_emails_list
+
+
+class SSOHost(Host):
```

Comment:
@JacobCallahan I'm starting to think `robottelo.hosts` should be a package, not a module.

Reply:
oooff that's something we can think about later...